core/app/models/concerns/spree/active_storage_attachment.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Spree
+  module ActiveStorageAttachment
```

Comment:
Could you explain why you added a second module for active storage support? Why not have these methods in the other module as well?

Reply:
This module is included by both AS implementations (Image and Taxon) and is an extraction of repeated code. The call between extracting/DRYing and leaving the duplication was, admittedly, close. But in the end I ended up refactoring in order to avoid the risk of leaving one of the two behind when fixing stuff.

Reply:
Since ActiveStorage is meant to add attachments to any model I like the idea here.